app/client/components/DetailView.js

```diff
@@ -75,18 +77,56 @@ function DetailView(gristDoc, viewSectionModel) {
 
   //--------------------------------------------------
   // Set up DOM event handling.
+  this.lastFieldIdsSelected = [null, null];
+
+  this._canSingleClick = function(field) {
```

Comment:
Can you move this under DetailView.prototype at the bottom of this file? Thanks.